### ISSUE TITLE
Allow for importing custom SVGs

### ISSUE THREAD

**Issue by Dionicio3** [programminghoch10/MobileIconChanger#8]
Hey, I was wondering if it would be possible to import your own SVGs to be used as data icons. 

Also, unrelated but, the readme has a copy paste error that references your CameraControl module

**Comment by programminghoch10** [comment]
> Hey, I was wondering if it would be possible to import your own SVGs to be used as data icons.

No, it isn't, as the SVGs have to be converted to androids drawable format in android studio first.
Even if it would be possible, it's just not worth the effort.
You can easily add icons yourself though and compile yourself.

> Also, unrelated but, the readme has a copy paste error that references your CameraControl module

Shh, thats surreptitious advertising
